apps/storybook/src/Breadcrumbs.test.ts

```diff
@@ -20,7 +20,8 @@ describe('Breadcrumbs', () => {
       cy.visit('iframe', { qs: { id } });
 
       if (testName === 'Custom Overflow Dropdown') {
-        cy.get('.iui-button').eq(1).click();
+        cy.get('[aria-label="Dropdown with more breadcrumbs"').click();
+        cy.wait(1000); // wait for dropdown to open?
```

Comment:
It seems like despite all these changes, it still fails 2 out of 3 times because the dropdown doesn't even open.

If it continues to be flaky, I think we should exclude this particular story.